src/test/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/AddAttendeeIntTest.kt

```diff
@@ -154,12 +147,9 @@ class AddAttendeeIntTest : IntegrationTestBase() {
   @Test
   fun `201 created - attendee added NOMIS`() {
     val prisonNumber = givenValidPrisonNumber("N1234NM")
-    val review = transactionTemplate.execute {
```

Comment:
good job :)  nice to see this improvement - reducing complexity and improved code quality by utilising jpa capabilities